package.json

```diff
@@ -39,59 +39,58 @@
   "license": "Apache-2.0",
   "dependencies": {
     "debug": "^4.1.1",
-    "extract-zip": "^2.0.0",
+    "extract-zip": "^2.0.1",
     "https-proxy-agent": "^5.0.0",
-    "jpeg-js": "^0.4.0",
-    "mime": "^2.4.4",
+    "jpeg-js": "^0.4.2",
+    "mime": "^2.4.6",
     "pngjs": "^5.0.0",
     "progress": "^2.0.3",
     "proxy-from-env": "^1.1.0",
     "rimraf": "^3.0.2",
-    "ws": "^6.1.0"
+    "ws": "^7.3.1"
   },
   "devDependencies": {
     "@babel/code-frame": "^7.10.4",
-    "@babel/core": "^7.10.3",
-    "@babel/preset-env": "^7.10.3",
-    "@babel/preset-typescript": "^7.10.1",
+    "@babel/core": "^7.11.4",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-typescript": "^7.10.4",
     "@types/babel__core": "^7.1.9",
-    "@types/debug": "0.0.31",
+    "@types/debug": "^4.1.5",
     "@types/extract-zip": "^1.6.2",
-    "@types/mime": "^2.0.1",
-    "@types/node": "^10.17.17",
-    "@types/pngjs": "^3.4.0",
+    "@types/mime": "^2.0.3",
+    "@types/node": "^10.17.28",
+    "@types/pngjs": "^3.4.2",
     "@types/progress": "^2.0.3",
-    "@types/proxy-from-env": "^1.0.0",
-    "@types/rimraf": "^2.0.2",
+    "@types/proxy-from-env": "^1.0.1",
+    "@types/rimraf": "^3.0.0",
     "@types/source-map-support": "^0.5.3",
     "@types/stack-utils": "^1.0.1",
-    "@types/ws": "^6.0.1",
+    "@types/ws": "7.2.6",
     "@typescript-eslint/eslint-plugin": "^3.10.1",
     "@typescript-eslint/parser": "^3.10.1",
     "colors": "^1.4.0",
     "commander": "^6.0.0",
-    "commonmark": "^0.28.1",
-    "cross-env": "^5.0.5",
-    "electron": "^9.0.0-beta.24",
+    "commonmark": "^0.29.1",
+    "cross-env": "^7.0.2",
+    "electron": "^9.2.1",
     "eslint": "^7.7.0",
     "eslint-plugin-notice": "^0.9.10",
-    "esprima": "^4.0.0",
-    "expect": "^26.3.0",
-    "formidable": "^1.2.1",
+    "expect": "^26.4.2",
+    "formidable": "^1.2.2",
     "ncp": "^2.0.0",
-    "node-stream-zip": "^1.8.2",
+    "node-stream-zip": "^1.11.3",
```

Comment:
Updating the minor/patch components should have no effect here. Maybe when bumping major version component for the deps we should rewrite them as "1.x" instead of "^1.11.3" so that it's more clear that we don't have any constraints on the minor version.

Reply:
Yep, it was mostly about the minor bumps. 👍